File: backend/src/main/java/nl/wilcokas/luckystackworker/model/PSF.java
package nl.wilcokas.luckystackworker.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PSF {
  private double airyDiskRadius;
  private double seeingIndex;
  private double diffractionIntensity;
  private PSFType type;
}

File: backend/src/main/kotlin/nl/wilcokas/luckystackworker/model/PSF.kt
package nl.wilcokas.luckystackworker.model

data class PSF(
    var airyDiskRadius: Double = 0.0,
    var seeingIndex: Double = 0.0,
    var diffractionIntensity: Double = 0.0,

    var airyDiskRadiusGreen: Double = 0.0,
    var seeingIndexGreen: Double = 0.0,
    var diffractionIntensityGreen: Double = 0.0,

    var airyDiskRadiusBlue: Double = 0.0,
    var seeingIndexBlue: Double = 0.0,
    var diffractionIntensityBlue: Double = 0.0,

    var type: PSFType? = null,
)
